proto/osmosis/gamm/v1beta1/query.yml

```diff
@@ -0,0 +1,86 @@
+keeper:
+  path: "github.com/osmosis-labs/osmosis/v11/x/gamm"
+  struct: "Keeper"
+client_path: "github.com/osmosis-labs/osmosis/v11/x/gamm/client"
+queries:
+  Pool:
+    proto_wrapper:
+      default_values:
+        Req.poolId: "1"
+      query_func_name: "Pool" 
+      query_req_rep_name: "QueryPool"
```

Comment:
I'm a bit confused, what is this difference for?

Reply:
This is for functions like `EstimateSwapExactAmountOut` because its `function_name != funcReq/RespName` 
function name: EstimateSwapExactAmountOut
function response/request: QuerySwapExactAmountOutResponse/ QuerySwapExactAmountOutRequest

The template currently expects the function name to be the same with function response/request but there are functions like `EstimateSwapExactAmountOut` where its not the case. 

Ideally we would want to set a naming standard for that matches with what the template expects